lib/teiserver/tachyon_battle.ex

```diff
@@ -54,6 +40,24 @@ defmodule Teiserver.TachyonBattle do
     end
   end
 
+  @spec start_battle(Teiserver.Autohost.id(), T.match_id()) ::
+          {:ok, T.id(), pid()} | {:error, term()}
+  def start_battle(autohost_id, match_id) do
```

Comment:
I believe this function should be private to this module, I don't think it's used elsewhere and it's rather confusing to have to function with identical name. Perhaps also rename into `start_battle_process` or something like that.